app/src/main/java/uc/edu/klopfsea/groupscheduler/ui/main/AddEvent.kt

```diff
@@ -48,8 +48,8 @@ class AddEvent : Activity(), AdapterView.OnItemSelectedListener {
 
             // converts the entered time to text and prints
             val time = findViewById<EditText>(R.id.editTextTime).text.toString()
-            System.out.println(time)
-            System.out.println(timeSpecification)
+            println(time)
+            println(timeSpecification)
```

Comment:
Go one step further and use a log statement.  No one will see a println, and println does not have the concept of priority, like logs do.